jinahub/segmenters/VADSpeechSegmenter/vad_speech_segmenter.py

```diff
@@ -0,0 +1,135 @@
+__copyright__ = 'Copyright (c) 2020-2021 Jina AI Limited. All rights reserved.'
+__license__ = 'Apache-2.0'
+
+from pathlib import Path
+from typing import Optional, Dict
+
+import torch
+import torchaudio
+import numpy as np
+import librosa as lr
+from jina import Executor, Document, DocumentArray, requests
+from jina.excepts import BadDocType
+
+
+class VADSpeechSegmenter(Executor):
+    """
+     Segment the speech audio using Silero's Voice Activity Detector (VAD)
+
+    :param normalize: a bool to specify whether to normalize the audio
+     by the sample rate
+    :param dump: a bool to specify whether to dump the segmented audio
+    :param repo: default repo name of silero-vad
+    :param model: default model name of silero-vad
+
+    """
+
+    TARGET_SAMPLE_RATE = 16000
+
+    def __init__(
+        self,
+        normalize: bool = False,
+        dump: bool = False,
+        repo: str = 'snakers4/silero-vad',
+        model: str = 'silero_vad',
+        *args,
+        **kwargs,
+    ):
+        super().__init__(*args, **kwargs)
+        if model is None or repo is None:
+            raise ValueError('model and repo cannot be None')
```

Comment:
Test this error

Reply:
Addressed, please see [here](https://github.com/jina-ai/executors/pull/97/commits/26ae02fc9d1b12c0c381f0495f4f1ae1d900b677) for the change.